crates/ide_completion/src/render.rs

```diff
@@ -979,12 +990,123 @@ fn main() {
                         detail: "S",
                         relevance: CompletionRelevance {
                             exact_name_match: true,
-                            exact_type_match: false,
+                            exact_type_match: true,
                         },
                         ref_match: "&mut ",
                     },
                 ]
             "#]],
         )
     }
+
+    #[test]
+    fn suggest_deref() {
+        check(
```

Comment:
We should use a less verbose `check_relevance` test here (ther's already code for that)